rDNA/R/rDNA.R

```diff
@@ -1806,4 +1807,289 @@ print.dna_barplot <- function(x, trim = 30, attr = TRUE, ...) {
   x2$Entity <- sapply(x2$Entity, function(e) if (nchar(e) > trim) paste0(substr(e, 1, trim - 1), "*") else e)
   class(x2) <- "data.frame"
   print(x2)
+}
+
+#' Plot dna_barplot object.
+#'
+#' Plot barplots generated from dna_barplot.
+#'
+#' This function plots agreement and disagreement towards DNA Statements for
+#' different entities such as, e.g., "concept", "organization" or "person".
```

Comment:
Add a sentence stating that this plots `dna_barplot` objects generated by the `dna_barplot` function.